core-services/importExport/src/main/java/org/silverpeas/core/importexport/model/SilverPeasExchangeType.java

```diff
@@ -24,102 +24,87 @@
 /*
  * Created on 24 janv. 2005
  *
- * To change the template for this generated file go to
- * Window&gt;Preferences&gt;Java&gt;Code Generation&gt;Code and Comments
  */
 package org.silverpeas.core.importexport.model;
 
-import org.silverpeas.core.importexport.admin.ComponentsType;
-import org.silverpeas.core.node.importexport.NodeTreesType;
-import org.silverpeas.core.pdc.pdc.importexport.PdcType;
+import org.silverpeas.core.admin.component.model.ComponentInst;
+import org.silverpeas.core.node.importexport.NodeTreeType;
+import org.silverpeas.core.pdc.pdc.importexport.AxisType;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
 
 /**
  * @author tleroi
  */
+@XmlRootElement(name = "SilverpeasExchange", namespace = "http://www.silverpeas.org/exchange")
+@XmlAccessorType(XmlAccessType.NONE)
 public class SilverPeasExchangeType {
 
+  @XmlAttribute
   private String targetComponentId;
+  @XmlAttribute
   private boolean usePOI = true;
-  private PublicationsType publicationsType;
-  private RepositoriesType repositoriesType;
-  private NodeTreesType nodeTreesType;
-  private PdcType pdcType;
-  private ComponentsType componentsType;
-
-  /**
-   * @return
-   */
-  public String getTargetComponentId() {
-    return targetComponentId;
+  @XmlElementWrapper(name = "publications")
+  @XmlElement(name = "publication", namespace = "http://www.silverpeas.org/exchange")
+  private List<PublicationType> publicationsType;
+  @XmlElementWrapper(name = "repositories")
+  @XmlElement(name = "repository")
+  private List<RepositoryType> repositoriesType;
+  @XmlElementWrapper(name = "topicTrees")
+  @XmlElement(name = "topicTree")
+  private List<NodeTreeType> nodeTreesType;
+  @XmlElementWrapper(name = "pdc")
+  @XmlElement(name = "axis")
+  private List<AxisType> pdcType;
+  @XmlElementWrapper(name = "components")
+  @XmlElement(name = "component")
+  private List<ComponentInst> componentsType;
+
+  public SilverPeasExchangeType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)